README.md

```diff
@@ -40,10 +41,11 @@ To generate multiple clients, specify multiple `<execution>` sections.
 | Parameter Name | Description |
 |:---------------|:------------|
 | outputPath | Location of generated classes (defaults to `${project.build.directory}/generated-sources/swagger-clients`) |
+| language | Which language to generate (defaults to `java`. Valid values are: `java`, `scala`) |
```

Comment:
I'm still feeling blah about this.  Might severely confuse/annoy people to update and then find that things are not working right afterward with the default change.  But at the same time I agree that java _should_ be the default for the maven plugin.  Maybe we could leave it as scala, but if no language is specified print a WARNING that says they really need to specify the language?  And then a few versions down the line change it for reals.

Reply:
I'd be in favor of that -- possibly adding a phase where there is no default, requiring a change, before changing to `java` being the default. I'll make that change.

Reply:
Resolved.